### PR TITLE
#164069263 Create react app

### PR DESCRIPTION
**Pull request by nyabongoedgar**
**What does this PR do?**
- Creates an empty react app

**Description of the tasks to be completed?**

-  Have an empty react app, which can take in components and can be run the **npm start** command.

**How should this be manually tested?**

- Clone repo at https://github.com/andela/ah-groot-frontend.git, check out to ch-create-react-app-164069263 and run in your terminal " **npm start** "

**What are the relevant pivot tracker stores?**

- #164069263

### CODE REVIEW

**Review by NLSanyu — APPROVED** [review]

**Comment by akram256** [comment]
@nyabongoedgar , I have realized you only have @NLSanyu as the only reviewer, Could  you please add more reviews including @huxaiphaer. Thanks. Otherwise great work

**Review by akram256 — APPROVED** [review]
Nice work

**Review by walimike — APPROVED** [review]

**Comment by walimike** [comment]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -10,7 +10,8 @@ pids
 *.pid
 *.seed
 *.pid.lock
-
+#eliminating package.lock
```
Comment (huxaiphaer) [inline .gitignore]: What does this mean?

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// import React, { Component } from "react";
```
Comment (huxaiphaer) [inline src/components/App.js]: Why are you still having commented code, is this still useful? If the file isn't useful just remove it.

File: src/components/App.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+// import '../styles/App.css';
+
+// class App extends Component {
+//     render() {
```
Comment (Stanley-Okwii) [inline src/components/App.js]: Why is this file commented out? please work on this.

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const App = () => (
+  <div>
+      <h1>Hello world!!</h1>
+  </div>
+ )
+ ReactDOM.render(<App/>, document.getElementById('root'));
```
Comment (Stanley-Okwii) [inline src/index.js]: Add a new line at the end of this file

**Review by Stanley-Okwii — CHANGES_REQUESTED** [review]
Good work, please work on those changes!!

File: src/index.js (hunk 4 above)
Comment (huxaiphaer) [inline src/index.js]: You are calling `<App/>` component yet you commented it out, can you throw more light on this please ?

File: src/components/App.js (hunk 3 above)
Comment (nyabongoedgar) [reply src/components/App.js]: I have worked on it by uncommenting the contents of the file

File: src/index.js (hunk 4 above)
Comment (nyabongoedgar) [reply src/index.js]: New line added

Comment (nyabongoedgar) [reply src/index.js]: I have uncommented the code contained in the App component.

File: src/components/App.js (hunk 2 above)
Comment (nyabongoedgar) [reply src/components/App.js]: Comments have been removed

File: .gitignore (hunk 1 above)
Comment (nyabongoedgar) [reply .gitignore]: I have deleted it.

**Review by Stanley-Okwii — APPROVED** [review]

**Comment by huxaiphaer** [comment]
LGTM

**Pull request merged**